lib/worker/lib/workerLib.ts

```diff
@@ -231,10 +231,16 @@ export class Parent extends RequesterResponder {
     /** start worker */
     startWorker(childJsPath: string, terminalError: (e: Error) => any, customArguments: string[]) {
         try {
+            var spawnEnv = process.env;
+            spawnEnv['ATOM_SHELL_INTERNAL_RUN_AS_NODE'] = '1';
```

Comment:
Oh yes, oops! I will fix that.

Do you have an existing way of checking the operating system?